admin-ui/src/components/settings/settings-section-card.tsx

```
@@ -0,0 +1,50 @@
+import { forwardRef, type ReactNode } from "react"
+import { BorderBeam } from "@/components/ui/border-beam"
```

ESLint

[error] Missed spacing between "react" and "@/components/ui/border-beam".

(perfectionist/sort-imports)